content/docs/command-reference/import-db.md

```diff
@@ -0,0 +1,194 @@
+profiles. No additional configuration is necessary if you have setup dbt
+already. However, you can use `--profile`/`--target` options to override those
+settings. Those can be set in
+[`dbt`](/doc/user-guide/project-structure/configuration#dbt) config too.
+
+You can also reuse _dbt_'s connection profiles to run sql query, by specifying a
+`--profile` and/or `--target`.
+
+```dvc
+dvc import-db --sql 'select * from table' --profile jaffle_shop
+```
+
+If you are inside a _dbt_ repository, dvc can discover a default connection
+profile setup for your _dbt_ project and use it to run SQL query. No additional
+configuration is necessary in this case.
+
+## Installing database drivers
+
+DVC does not come preinstalled with all the drivers for databases, you’ll need
+to install the required packages for the database you want to use.
+
+Some of the recommended packages are shown below, with their expected connection
+strings:
+
+| **Database**        | **PyPI package**                  | **Connection String**                                                                               |
+| ------------------- | --------------------------------- | --------------------------------------------------------------------------------------------------- |
+| **Amazon Redshift** | `sqlalchemy-redshift`             | `redshift+psycopg2://{username}:{password}@{aws_endpoint}:5439/{database_name}`                     |
+| **Big Query**       | `pip install sqlalchemy-bigquery` | `bigquery://{project_id}`                                                                           |
+| **Databricks**      | `databricks-sql-connector`        | `databricks://token:{token}@{hostname}:{port}/{database}?http_path={http_path}`                     |
+| **MySQL**           | `mysqlclient`                     | `mysql://{username}:{password}@{hostname}/{database_name}`                                          |
+| **Oracle**          | `cx_Oracle`                       | `oracle://{username}:{password}@{hostname}/{database_name}`                                         |
+| **PostgreSQL**      | `psycopg2`                        | `postgresql://{username}:{password}@{hostname}/{database_name}`                                     |
+| **Snowflake**       | `snowflake-sqlalchemy`            | `snowflake://{user}:{password}@{account}.{region}/{database}?role={role}&amp;warehouse={warehouse}` |
+| **SQLite**          | -                                 | `sqlite://path/to/file.db?check_same_thread=false`                                                  |
+| **SQL Server**      | `pyodbc`                          | `mssql+pyodbc://{username}:{password}@{hostname}:{port}/{database_name}`                            |
+| **Trino**           | `trino`                           | `trino://{username}:{password}@{hostname}:{port}/{catalog}`                                         |
+
+DVC uses sqlalchemy internally. So DVC should support any SQL databases that
```

Comment:
I have only tested with `sqlite`, `postgres`, and `snowflake`. But it should probably work with all mentioned here (and more).